### PR TITLE
Upgrading Brunton Test

### PR DESCRIPTION
**Pull request by marcinpaluch1994**
Auxiliary plot
Possibility to evaluate arbitrary networks (prefix network name with "S:" for "Simple evaluation")
Removing not ready changes of Mehdi from Master

### CODE REVIEW

File: src/SI_Toolkit/Functions/TF/Training.py [hunk 1]
```diff
@@ -39,16 +37,17 @@ def train_network_core(net, net_info, training_dfs_norm, validation_dfs_norm, te
...
-    #     optimizer=keras.optimizers.Adam(0.001)
+    #     loss="mse",
+    #     optimizer=keras.optimizers.Adam(a.lr)
     # )
+    from SI_Toolkit.Functions.TF.Loss import loss_msr_sequence_customizable
+    net.compile(
+        loss=loss_msr_sequence_customizable(wash_out_len=a.wash_out_len,
+                                            post_wash_out_len=a.post_wash_out_len,
+                                            discount_factor=1.0),
+        optimizer=keras.optimizers.Adam(0.001)
```
Comment (frehe) [inline src/SI_Toolkit/Functions/TF/Training.py:49]: Why make learning rate a constant here? Was `a.lr` before

Comment (marcinpaluch1994) [reply src/SI_Toolkit/Functions/TF/Training.py:49]: Thank you. I revert this change now.
